src/serac/physics/state/finite_element_vector.hpp

```diff
@@ -51,10 +62,14 @@ class FiniteElementVector : public mfem::HypreParVector {
      * Defaults to scalar valued spaces.
      */
     int vector_dim = 1;
+
     /**
-     * @brief The FECollection to use - defaults to an H1_FECollection
+     * @brief Enum deoniting type of basis functions to use
```

Comment:
denoting maybe?
```suggestion
     * @brief Enum denoting type of basis functions to use
```

maybe?